frontend/src/component/project/Project/ProjectStatus/ProjectLifecycleSummary.tsx

```diff
@@ -165,6 +167,10 @@ export const ProjectLifecycleSummary = () => {
         );
     };
 
+    const totalArchivedStat = () => {
+        const count = data?.lifecycleSummary.archived.last30Days ?? 0;
+        return `${count} ${count === 1 ? 'flag' : 'flags'} archived`;
+    };
```

Comment:
this can be a variable, not a function

Reply:
It *can* be, but I'm not sure what you get from it. doing it as a function gives you scope to name variables and separate it from other stuff. I don't mind, though 🤷🏼

Reply:
Scoping is a for of an abstraction - to me it makes this code less declarative, less "linear" to read if you need to understand everything that's going on, instead of trusting the function name